unity/Assets/Scripts/ArmAgentController.cs

```diff
@@ -20,6 +20,28 @@ public class ArmAgentController : PhysicsRemoteFPSAgentController {
             return arm;
         }
 
+        public void MoveArmRelative(
```

Comment:
as mentioned in my review of #803, I'd like a quick comment summary for this action to make documentation upkeep easier.

Reply:
I have added this documentation.